tests/playwright-test/expect-poll.spec.ts

```diff
@@ -232,3 +232,16 @@ test('should show intermediate result for poll that spills over test time', asyn
   expect(result.output).toContain('Expected: 2');
   expect(result.output).toContain('Received: 3');
 });
+
+test('should propagate promise rejections', { annotation: { type: 'issue', description: 'https://github.com/microsoft/playwright/issues/32256' } }, async ({ runInlineTest }) => {
```

Comment:
Let's add a second test that would have the predicate `async () => { throw 'String value'; }`.

Reply:
done in 6b07b2287b2bea98fbfdba1a0611952183393827